phac-nml/irida-next
# [ENHC0010042] Add new file format labels for attachments

## What does this PR do and why?
Added new labels for when a file is uploaded. The label `text` is displayed for file formats (txt, csv, tsv, rtf) and the label `spreadsheet` is displayed for file formats (xls, xlsx).
Fixes #661.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/233c7e23-588a-4b2f-8cbe-b6313d78e7f7)

## How to set up and validate locally
1. Login to irida-next
2. Create a new project.
3. Create a new sample within the project.
4. Upload files with `txt`, `csv`, `tsv`, & `rtf` extensions.
5. Verify that `text` labels are being displayed under the `format` column. 
6. Upload files with `xls` & `xlsx` extensions.
7. Verify that `spreadsheet` labels are being displayed under the `format` column. 

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
